vizro-core/tests/unit/vizro/models/_components/conftest.py

```diff
@@ -0,0 +1,50 @@
+"""Fixtures to be used in components related tests."""
+import plotly.graph_objects as go
+import pytest
+from dash import dcc
+
+import vizro.plotly.express as px
+from vizro.models._components.graph import create_empty_fig
+
+
+@pytest.fixture
+def standard_px_chart_with_str_dataframe():
```

Comment:
I think the idea is that if the fixture is used only in one file, we can also put it in that file, so this one would go into `test_graph.py`, likewise for the others potentially

Reply:
All of the fixtures here would go into test_graph 👍